fix(Toggle): Convert rem to px-rem()

goopaloop opened the pull request:
## Overview

This PR cleans up the rem sizing for the track and thumb (some of the decimals go out 2+ places). Converting rem to the px-rem() function as per Aaron's suggestion in [PR 885](https://github.com/Codecademy/client-modules/pull/885)


### PR Checklist

- [ ] Related to Abstract designs:
- [x] Related to JIRA ticket: [GM-64](https://codecademy.atlassian.net/browse/GM-64)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

JoshuaKGoldberg commented on packages/gamut/src/Toggle/index.tsx:
```diff
@@ -37,6 +39,7 @@ export class Toggle extends Component<ToggleProps, {}> {
           id={label}
           disabled={disabled}
           onChange={onChange}
+          onClick={onClick}
```
What about `onChange` isn't doing it for you? I would think it enough?

goopaloop replied:
wait, you're right I can just add event tracking in `onChange`. I was previously using `onChange` to flip a boolean, and thought I needed a separate `onClick` prop for the event tracking function.

I can remove the `onChange` parts of this ticket and just make the px-rem() changes.

goopaloop commented:
@JoshuaKGoldberg i added `px` to the number in px-rem() to make it more explicit -- example: `px-rem(5)` is now `px-rem(5px)`. and also changed a couple values as requested by the design gods for the small toggle we are building

codecademydev commented:
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.6-alpha.c27975.0<br/>@codecademy/gamut@12.2.5-alpha.c27975.0<br/></pre><!--LERNA_VERSION_COMMENT-->

codecademydev commented:
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f2c6ec528dfa601716b7fe9--gamut-preview.netlify.app>https://5f2c6ec528dfa601716b7fe9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f2c6ec528dfa601716b7fe9>Deploy Logs</a></p><!--ALPHA_DEPLOY_COMMENT-->
